server/services/licenceService.ts

```diff
@@ -149,6 +149,14 @@ export default class LicenceService {
     return this.licenceApiClient.createLicence(licence, user)
   }
 
+  private async getCRONumber(prisonerNumber: string, user: User): Promise<string> {
```

Comment:
I think a name like `getCroNumberFromNomis` would be more appropriate, not only for TS-friendly casing but also to show it's coming from NOMIS

Reply:
Also very small point that's more a preference than a rule, I'd prefer this to be towards the bottom of the file, to keep the `private` methods together and also to keep the regularly-used methods (like `getLicence`) closer to the top for easy of reading